learn/advanced/storage.md

```diff
@@ -71,3 +71,9 @@ These metrics are highly dependent on the machine that is running Meilisearch. R
 It is important to note that **there is no reliable way to predict the final size of a database**. This is true for just about any search engine on the market—we're just the only ones saying it out loud.
 
 Database size is affected by a large number of criteria, including settings, relevancy rules, use of facets, the number of different languages present, and more.
+
+## Soft deletion
+
+Meilisearch renders deleted documents inaccessible to all users but does not immediately remove them from the database. This is a common optimization technique called "soft deletion". Soft deleted documents are permanently deleted during a later update, depending on your index size and the available disk space. It might be important to check how soft deletion interacts with data retention legislation relevant to your application.
```

Comment:
Should soft deletion be in quotes?

Reply:
It was intentional, but I'm not 100% convinced. The idea is to bring attention to "soft deletion" as a title instead of an action (like, "Meilisearch soft deletes documents" vs "the name of this technique is 'soft deletion'"). In essence, what grammarly defines as ["using words as words"[(https://www.grammarly.com/blog/quotation-marks/).

What do you think?

Reply:
We currently use quotes when referring to button text, field names, etc. Might be a longer discussion, but we could use italics instead

Reply:
Hmmm…not sure I like italics. I'll just leave it without for now.